devicetypes/keen-home/keen-home-smart-vent.src/keen-home-smart-vent.groovy

```diff
@@ -477,47 +186,25 @@ def configure() {
     // enrolls with default periodic reporting until newer 5 min interval is confirmed
     sendEvent(name: "checkInterval", value: 2 * 10 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
 
-    // get ZigBee ID by hidden tile because that's the only way we can do it
-    setZigBeeIdTile()
-
-    def configCmds = [
-        // bind reporting clusters to hub
-            //commenting out switch cluster bind as using wrapper onOffConfig of zigbee class
-        //"zdo bind 0x${device.deviceNetworkId} 1 1 0x0006 {${device.zigbeeId}} {}", "delay 500",
-        "zdo bind 0x${device.deviceNetworkId} 1 1 0x0008 {${device.zigbeeId}} {}", "delay 500",
-        "zdo bind 0x${device.deviceNetworkId} 1 1 0x0402 {${device.zigbeeId}} {}", "delay 500",
-        "zdo bind 0x${device.deviceNetworkId} 1 1 0x0403 {${device.zigbeeId}} {}", "delay 500",
-        "zdo bind 0x${device.deviceNetworkId} 1 1 0x0001 {${device.zigbeeId}} {}", "delay 500"
-
-        // configure report commands
-        // zcl global send-me-a-report [cluster] [attr] [type] [min-interval] [max-interval] [min-change]
-
-        // report with these parameters is preconfigured in firmware, can be overridden here
-        // vent on/off state - type: boolean, change: 1
-        // "zcl global send-me-a-report 6 0 0x10 5 60 {01}", "delay 200",
-        // "send 0x${device.deviceNetworkId} 1 1", "delay 1500",
-
-        // report with these parameters is preconfigured in firmware, can be overridden here
-        // vent level - type: int8u, change: 1
-        // "zcl global send-me-a-report 8 0 0x20 5 60 {01}", "delay 200",
-        // "send 0x${device.deviceNetworkId} 1 1", "delay 1500",
+    def cmds = [
+            zigbee.temperatureConfig(30, 300) +
+                    zigbee.addBinding(zigbee.ONOFF_CLUSTER) +
+                    zigbee.addBinding(zigbee.LEVEL_CONTROL_CLUSTER) +
+                    zigbee.configureReporting(PRESSURE_MEASUREMENT_CLUSTER, 0x0020, DataType.UINT32, 600, 21600, 0x010000, [mfgCode: MFG_CODE]) +
+                    zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021, DataType.UINT8, 600, 21600, 0x01) // battery precentage
+    ]
 
-        // report with these parameters is preconfigured in firmware, can be overridden here
-        // temperature - type: int16s, change: 0xA = 10 = 0.1C
-        // "zcl global send-me-a-report 0x0402 0 0x29 60 60 {0A00}", "delay 200",
-        // "send 0x${device.deviceNetworkId} 1 1", "delay 1500",
+    return delayBetween(cmds) + refresh()
```

Comment:
`delayBetween` should not be necessary. There are already delays added by the `zigbee` library. Also the `refresh` commands should be sent first so the current state is reflected more quickly.

Reply:
I think this should be left as `refresh() + cmds`